src/index.js

```diff
@@ -4,24 +4,29 @@ const fetchStatuCakeData = require('./statuscake');
 const server = require('./server');
 const storage = require('./statuscakedata');
 
-const regex = /\s?([-])\s?/;
+const regexp = config.statuscake.regex;
 
-const getTagsFromTitle = Title => Title.split(regex);
+const getTagsFromTitle = title => title.match(new RegExp(regexp));
+const tags = [
+  (config.statuscake.customer),
```

Comment:
🙅‍♂ Nop, this tags array is not generic. Tag list should be declared in the config 😇

Reply:
Exact 🐙